src/Statistics.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  color: #383854;
+  margin: 4rem 0 2rem 0;
+  font-size: 2rem;
```

Comment:
Font size on a  none text HTML?
your not giving anything font-size at the moment